web-dev-server.config.js

```diff
@@ -0,0 +1,134 @@
+import { hmrPlugin, presets } from '@open-wc/dev-server-hmr';
```

Comment:
**fix:** you can remove this file, I think it's because your branch started before we migrated to Vite but it's no longer needed :wink: